Doc/tools/extensions/pyspecific.py

```diff
@@ -253,7 +253,7 @@ def run(self):
             text = 'The NEWS file is not available.'
             node = nodes.strong(text, text)
             return [node]
-        content = issue_re.sub(r'`\1ssue #\2 <https://bugs.python.org/\2>`__',
+        content = issue_re.sub(r'`\bpo-\2 <https://bugs.python.org/\2>`__',
```

Comment:
That's on purpose as the first capture group is the prefix which doesn't matter. Probably should change the regex to use a non-capture group.